src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/movements/MovementsService.kt

```diff
@@ -195,6 +196,7 @@ class MovementsService(
       ?: throw NotFoundException("Scheduled temporary absence with eventId=$eventId not found for offender with nomsId=$offenderNo")
 
     return scheduledAbsence.toSingleResponse()
+      .also { log.info("Returning scheduled outbound movement: eventId=$eventId, nomsId=$offenderNo, eventStatus=${it.eventStatus}, inboundEventStatus=${it.inboundEventStatus}, auditUser=${it.audit.auditUserId}, auditTimestamp=${it.audit.auditTimestamp}, auditModule=${it.audit.auditModuleName}") }
```

Comment:
The event status / inbound status don't appear to reflect what is on the NOMIS table when processing the triggered event. This doesn't seem to be possible. Add logging to try and work out what is going on.